src/test/specs/ExpectNoAttributeValueSpec.ts

```diff
@@ -18,4 +18,14 @@ describeCommon('expectNoAttributeValue', () => {
   it('Exact match error thrown', () => {
     expect(() => BrowserUtils.expectNoAttributeValue(EMPTY_DIV, 'align', 'center')).to.throw(Error);
   });
+
+  it('Case Element not found.', () => {
```

Comment:
suggestion for name: "not existing element"

Reply:
changed to "Element not exists"